src/components/structures/TimelinePanel.tsx

```diff
@@ -994,7 +994,8 @@ class TimelinePanel extends React.Component<IProps, IState> {
                     if (e.errcode === 'M_UNRECOGNIZED' && lastReadEvent) {
                         if (
                             !sendRRs
-                            && !cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable")
+                            && !await cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable")
+                            && !await cli.isVersionSupported("v1.4")
```

Comment:
```suggestion
                            && !(await cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable"))
                            && !(await cli.isVersionSupported("v1.4"))
```

I'm paranoid about the runtime doing the wrong thing, sorry

Reply:
Understandable